tools/isoplot/isoplot.xml

```diff
@@ -0,0 +1,155 @@
+        <!--param name='run_name' type='text' label='Run name' /-->
+        <param name='format' type='select' label='Plot format'>
+            <option value='pdf'>pdf</option>
+            <option value='jpeg'>jpeg</option>
+            <option value='png'>png</option>
+            <option value='svg'>svg</option>
+            <option value='html'>html</option>
+        </param>
+        <param name='value' type='select' label='Data to plot' >
+            <option value='corrected_area'>Corrected Area</option>
+            <option value='isotopologue_fraction'>Isotopologue Fraction</option>
+            <option value='mean_enrichment'>Mean Enrichment</option>
+        </param>
+        <section name='SP' title='Select static plots to generate'>
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' argument='-bp'
+                   optional='true'/>
+            <param type='boolean' name='generate_meanplot' label='Flag to generate meaned barplots' argument='-mb'
+                   optional='true'/>
+            <!--<param type='boolean' name='generate_areaplot' label='Flag to generate stacked area plots'
+                   argument='-sa' optional='true'/>/-->
+            <param type='boolean' name='generate_heatmap' label='Flag to generate heatmap' argument='-hm'
+                   optional='true'/>
+            <param type='boolean' name='generate_clustermap' label='Flag to generate clustermap' argument='-cm'
+                   optional='true'/>
+        </section>
+        <section name='IP' title='Select interactive plots to generate'>
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' argument='-IB'
+                   optional='true'/>
+            <param type='boolean' name='generate_meanplot' label='Flag to generate meaned barplots' argument='-IM'
+                   optional='true'/>
+           <!-- <param type='boolean' name='generate_areaplot' label='Flag to generate stacked area plots' argument='-IS'
+                   optional='true'/>-->
+            <param type='boolean' name='generate_heatmap' label='Flag to generate heatmap' argument='-HM'
+                   optional='true'/>
+        </section>
+        <param type='boolean' name='stack' label='Flag to unstack bars in barplots' argument='-s' optional='true'/>
+        <param type='boolean' name='verbosity' label='Flag to get debug information' argument='-v' optional='true'/>
+        <param type='boolean' name='annotation' label='Flag to add annotations on the maps' argument='-a'
+               optional='true'/>
+    </inputs>
+    
+    <outputs>
+        <data name='output1' format='csv' label='Formatted data used for plotting' />
+        <data name='output2' format='zip' label='Resulting plots' />
+        <data name='output3' format='txt' label='Run Info' />

```

Comment:
should that be only outputted if verbose is set?

Reply:
Yes but is it useful for the user by default? Should we keep the history clean by default?

Reply:
I will think it through and make changes to the wrapper on the next release!

Reply:
Agree with @bgruening.
If you need help about the dataset collection, you can give us an example of the contain of a zip file.